README.md

```diff
@@ -75,6 +76,36 @@ GET /manifests
 GET /manifests/<name>
 ````
 
+## Determine whether partial file uploads are supported
+````
+OPTIONS /files
+````
+This returns the following header and body:
+````
+Server: ....
+Allow: GET,HEAD,POST,OPTIONS,TRACE
+Content-Type: Application/octet-binary
+
+{
```

Comment:
Guessing someone's done this sort of format before. Other than WSDL. May be worth a look around to see what that format looked like.